src/components/dev-hub/green-bulleted-list.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { P } from '~components/dev-hub/text';
+import { size } from '~components/dev-hub/theme';
+
+const GreenBullet = styled('ul')`
+    color: ${({ theme }) => theme.colorMap.darkGreen};
+    list-style-type: circle;
+`;
+
+const ListItemWithSpacing = styled('li')`
+    margin-bottom: ${size.default};
+`;
+
+export const DefaultBulletText = styled(P)`
+    color: ${({ theme }) => theme.colorMap.devWhite};
+    margin-bottom: 0;
+`;
+
+const GreenBulletedList = ({
+    bulletTextComponent = DefaultBulletText,
+    children,
+    ...props
+}) => {
+    const BulletText = bulletTextComponent;
+    return (
+        <GreenBullet {...props}>
+            {children.map(content => (
+                <ListItemWithSpacing key={content}>
+                    <BulletText>{content}</BulletText>
```

Comment:
I wanted to reuse this GreenBulletedList component but wanted to slightly modify it so that I can style BulletText with `P2` instead of `P`. I did this by modifying the component in a way that would allow me to abstract the BulletText component. I also moved this file from components/pages/educators to components/dev-hub. Thought it made more sense now that I'm using it in other places. Please lmk what you think about this approach!

Reply:
I think we definitely should reuse this! Awesome thought